### ISSUE TITLE
Focus ring around combo box in many dialogs is broken

### ISSUE THREAD

**Issue by jonpalmisc** [Vector35/binaryninja-api#4787]
Notice lack of focus ring around the main input field. Affects type dialog, go to offset dialog, etc.

Binary Ninja 3.6.4685-dev

<img width="592" alt="image" src="https://github.com/Vector35/binaryninja-api/assets/41548458/69e8aca9-0615-4739-908c-7e5602813293">

**Comment by negasora** [comment]
This was fixed in `3.6.4721-dev`
